satish364.md

```diff
@@ -0,0 +1,4 @@
+This is my new file. 
+
+* Name: Satish
+*Location: San Diego, CA
```

Comment:
Looks like you’re missing the space after the asterisk here, which makes location not render as a bullet. Should be:

```
* Location
```